helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -184,8 +184,11 @@ receivers:
         config:
           extraDimensions:
             metric_source: kubernetes-scheduler
-          port: 10251
+          skipVerify: true
+          port: 10259
           type: kubernetes-scheduler
+          useHTTPS: true
+          useServiceAccount: true
```

Comment:
Looks like [this change](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md#changes-by-kind-17) is made default in k8s version 1.23. We might still have customers on older k8s version who will need a workaround to use the old config. How about we add a check to add the new config iff k8s version is >= 1.23?

Reply:
We don't lookup the Kubernetes version and use it in the Helm templates. I want to let this age out and delete this later to keep the scope small.

I want to refactor all of our control plane monitoring in the future and only maintenance it for now.